code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -118,20 +118,42 @@
 	return
 
 /obj/item/robot_module/standard
-	name = "standard robot module"
+	// if station is fine, assist with constructing station goal room, cleaning, and repairing cables chewed by rats
+	// if medical crisis, assist by providing basic healthcare, retrieving corpses, and monitoring crew lifesigns
+	// if eng crisis, assist by helping repair hull breaches
+	// if sec crisis, assist by opening doors for sec and providing a backup stunbaton on patrols
+	name = "generalist robot module"
 	module_type = "Standard"
+	subsystems = list(/mob/living/silicon/proc/subsystem_power_monitor, /mob/living/silicon/proc/subsystem_crew_monitor)
+	stacktypes = list(
+		/obj/item/stack/sheet/metal/cyborg = 50,
+		/obj/item/stack/cable_coil/cyborg = 50,
+		/obj/item/stack/rods/cyborg = 60,
+		/obj/item/stack/tile/plasteel = 20
+		)
 
 /obj/item/robot_module/standard/New()
 	..()
+	// sec
 	modules += new /obj/item/melee/baton/loaded(src)
```

Comment:
I'd rather we just do what TG has done here, with this set of items;

```
		/obj/item/assembly/flash/cyborg,
		/obj/item/reagent_containers/borghypo/epi,
		/obj/item/healthanalyzer,
		/obj/item/weldingtool/largetank/cyborg,
		/obj/item/wrench/cyborg,
		/obj/item/crowbar/cyborg,
		/obj/item/stack/sheet/metal/cyborg,
		/obj/item/stack/rods/cyborg,
		/obj/item/stack/tile/plasteel/cyborg,
		/obj/item/extinguisher,
		/obj/item/pickaxe,
		/obj/item/t_scanner/adv_mining_scanner,
		/obj/item/restraints/handcuffs/cable/zipties,
		/obj/item/soap/nanotrasen
```

And do away with the stun baton. I'd rather not de-factor have the standard module become secborg 2.0

Reply:
Comparing mine to the TG borg:
1) mine has a stunbaton but no cuffs, theirs has cuffs but no stunbaton. I'm inclined to think mine is better, as the stunbaton has non-sec uses (like letting the borg defend itself from carp and other NPCs). Whereas the sole purpose of zipties is detaining people, for sec or antag purposes.
2) theirs has mining items (mining scanner, pickaxe).... which mine doesn't. It isn't a big deal, I just feel like there's no point in a generalist borg going to mining. 
3) theirs has more engi gear (they get a largetank welder, and a wrench)
4) mine's injector has saline, in addition to the epi theirs has. So a little better at medical. 

Overall, I don't think TG's version is better. 
I'm okay with changes 2 and 3. Though I worry they'll make the borg a bit too strong.
My worry is with changes 1 and 4. 1 I feel strips the borg of its primary ability to defend itself from stuff like carp. 4 I feel makes it nigh-useless at doing anything medical.

Reply:
Fair on the injector.

I'd go ahead and add the engineering and mining gear.

The reason the baton is more egregious than the cablecuffs is it's pretty easy to baton and drag someone all the way to security by repeatedly applying the baton before they get unstunned. Chances are you aren't going to completely drain your own cell with 4-5 stun baton whack---but you can effectively keep someone stunned for over a full minute under those circumstances.

Reply:
Generalist borg loadout has been tweaked - it now gets cuffs instead of a baton, and gets engi/mining gear. I also gave it an ore satchel because without one it can't actually use its mining gear effectively.